CMakeLists.txt

```diff
@@ -457,7 +459,7 @@ endif()
 set(MFEM_TPLS OPENMP HYPRE BLAS LAPACK SuperLUDist METIS SuiteSparse SUNDIALS PETSC
     SLEPC MESQUITE MUMPS STRUMPACK AXOM FMS CONDUIT Ginkgo GNUTLS GSLIB NETCDF
     MPFR PUMI HIOP POSIXCLOCKS MFEMBacktrace ZLIB OCCA CEED RAJA UMPIRE ADIOS2
-    CUSPARSE MKL_CPARDISO AMGX CALIPER CODIPACK BENCHMARK PARELAG MPI_CXX)
+    CUSPARSE ROCSPARSE MKL_CPARDISO AMGX CALIPER CODIPACK BENCHMARK PARELAG MPI_CXX)
```

Comment:
Should this be updated to `HIPSPARSE` in light of https://github.com/mfem/mfem/pull/2784?

Reply:
I'm not sure if `libhipsparse.*` defines the same symbols as `librocsparse.*`. Note that the `ROCSPARSE` dependency includes also `librocrand.*` since hypre needs that too.

Reply:
Right, hipsparse is a wrapper for ROCSPARSE and CUSPARSE